samples/GetAddressService/Tables/Address.al

```diff
@@ -3,30 +3,24 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 // ------------------------------------------------------------------------------------------------
 
-table 50100 Address
+table 70051000 Address
```

Comment:
Is this change intended to be part of the pull request?

Reply:
No, this change shouldn't really be there :D. Actually, this change should not exist at all. Anyway I fixed it ^^ @StanislawStempin